frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -182,23 +182,33 @@ action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected br
 
 action.GitMachete.DiscoverAction.text=Discover Branch Layout
 action.GitMachete.DiscoverAction.description=Discover Branch Layout Tree
-action.GitMachete.DiscoverAction.cant-discover-layout-error-title=Can't discover repository layout
-action.GitMachete.DiscoverAction.cant-get-current-repository-error-title=Can't get current git repository
+action.GitMachete.DiscoverAction.cant-discover-layout-error-title=Can''t discover repository layout
```

Comment:
@mkondratek @micpiotrowski will it be still correct if we just forbid single `'` in the strings completely? :thinking:

Reply:
With single `'` only this is evaluated to no `'` at all (IIUC your question).

Reply:
Ok but can we ban the regex `[^']'[^']` completely in this file?

It'd be nice to answer the following questions:

1. is there any context where `''` (double single quote) will NOT render as `'`?
2. is there any context where `'` (single single quote) will  NOT render as `'`?

I suspect the 1=>NO && 2=>YES... so forbidding single single quote would be a good idea then :thinking:

Reply:
Also the comment `WE use single quotes for indication` in line 2 is slightly misleading, maybe:

```
# Action descriptions & texts - no formatting allowed (WE use double single quotes, which render as single single quotes for indication)
```

Reply:
The followed rule of thumb provided by @PawelLipski via Slack conversation should be fine:
* strings that have any `{N}` must NOT have `'`, but `''` instead
* strings that do NOT have any `{N}` are NOT meant to be passed through MessageFormat and hence must NOT have `''` but plain `'` instead

Reply:
These are detailed rules about `'` in `MessageFormat`:
```
Within a String, a pair of single quotes can be used to quote any arbitrary characters except single quotes. For example, pattern string "'{0}'" represents string "{0}", not a FormatElement. A single quote itself must be represented by doubled single quotes '' throughout a String. For example, pattern string "'{''}'" is interpreted as a sequence of '{ (start of quoting and a left curly brace), '' (a single quote), and }' (a right curly brace and end of quoting), not '{' and '}' (quoted left and right curly braces): representing string "{'}", not "{}".

A SubformatPattern is interpreted by its corresponding subformat, and subformat-dependent pattern rules apply. For example, pattern string "{1,number,$'#',##}" (SubformatPattern with underline) will produce a number format with the pound-sign quoted, with a result such as: "$#31,45". Refer to each Format subclass documentation for details.

Any unmatched quote is treated as closed at the end of the given pattern. For example, pattern string "'{0}" is treated as pattern "'{0}'".

Any curly braces within an unquoted pattern must be balanced. For example, "ab {0} de" and "ab '}' de" are valid patterns, but "ab {0'}' de", "ab } de" and "''{''" are not.
```
Source: https://docs.oracle.com/javase/7/docs/api/java/text/MessageFormat.html

Reply:
Ok, pls provide a script to enforce the two above rules